src/App.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { actions as todosActions } from './features/todos';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector((state => state.todos));
+  const { query } = useAppSelector(state => state.filter);
+  const { status } = useAppSelector(state => state.filter);
```

Comment:
It seems that you can combine it
```suggestion
  const { query, status } = useAppSelector(state => state.filter);
```